core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -607,7 +607,7 @@ private void syncExistingInodeMetadata(LockedInodePath inodePath, boolean skipLo
               builder.setGroup(group);
             }
             SetAttributeContext ctx = SetAttributeContext.mergeFrom(builder)
-                .setUfsFingerprint(ufsFingerprint);
+                .setUfsFingerprint(ufsFpParsed.serialize());
```

Comment:
Only serialize when we really need to. It's a shame we store the string instead of a Fingerprint proto.